src/pages/Join.css

```diff
@@ -82,43 +84,44 @@
   padding-top: 25px;
 }
 
-.carousel-container {
+/* .carousel-container {
   background-color: aqua;
```

Comment:
For all of the code that you commented out, if it is likely we would uncomment it and use it again at some point then it is fine to leave it commented. If we are not likely to use it again in the near future, then make sure to delete it. Too much commented code just makes everything messy.